tile-window.lisp

```diff
@@ -456,11 +456,10 @@ frame. Possible values are:
                                                 (window-height-inc window)))))
       (maximize-window window)))))
 
-(defcommand (unmaximize tile-group) () ()
+(defcommand (unmaximize tile-group) (&optional (window (current-window))) (:rest)
```

Comment:
Does this change the interactive call behaviour? Will it now ask for a window?

Reply:
No. Interactively it still just uses the current window; it doesn't ask for any input.